.travis.yml

```diff
@@ -5,21 +5,20 @@ matrix:
     # Linux Python 2
     - sudo: required
       language: python
+      python: "2.7"
       services:
         - docker
       env:
-        - "PYTHON=python2"
+        - "PYTHON=python"
 
     # Linux Python 3
     - sudo: required
       language: python
+      python: "3.4"
```

Comment:
Why 3.4, and not the latest version? Or just 3, then? (All in all, the different minor versions of Python will be tested inside the `cibuildwheel` tests anyway)

Reply:
According to travis-ci docs, https://docs.travis-ci.com/user/languages/python/#Specifying-Python-versions, a major.minor shall be provided
This is the python version running cibuildwheel driver, not the python used to build wheels (so, cibuildwheel itself is not tested with every minor python versions). If support is dropped for 3.3 for building wheels, I found somewhat logical not to use it either to run cibuildwheel and thus, use the next supported version which is python 3.4